addons/cert-manager/cert-manager.yaml

```
@@ -5,13 +5,11 @@ metadata:
   labels:
     kubeaddons.mesosphere.io/name: cert-manager
   annotations:
-    catalog.kubeaddons.mesosphere.io/addon-revision: "1.0.1-2"
-    appversion.kubeaddons.mesosphere.io/cert-manager: "1.0.1"
+    catalog.kubeaddons.mesosphere.io/addon-revision: "1.0.3-1"
+    appversion.kubeaddons.mesosphere.io/cert-manager: "1.0.3"
     docs.kubeaddons.mesosphere.io/cert-manager: "https://cert-manager.io/docs/release-notes/release-notes-1.0/"
     values.chart.helm.kubeaddons.mesosphere.io/cert-manager: "https://raw.githubusercontent.com/mesosphere/charts/c36a9b7/staging/cert-manager-setup/values.yaml"
```

need to update this otherwise lgtm! 
```suggestion
    values.chart.helm.kubeaddons.mesosphere.io/cert-manager: "https://raw.githubusercontent.com/mesosphere/charts/885f3af/stable/cert-manager-setup/values.yaml"
```